src/api/spec/bootstrap/features/webui/notifications_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'browser_helper'
+
+RSpec.feature 'Notifications', type: :feature, js: true do
+  RSpec.shared_examples 'updatable' do
+    scenario 'notifications' do
+      login user
+      visit path
+
+      expect(page).to have_content(title)
+      notification_field = find('.card-body h5', text: 'Package has failed to build').sibling('.form-inline')
+      ['Maintainer', 'Bugowner', 'Reader', 'Watching the project'].each do |label|
+        notification_field.check(label, allow_label_click: true)
+        expect(page).to have_css('#flash', text: 'Notifications settings updated')
```

Comment:
The flash appears on the first time a checkbox is checked, so if we don't close it before checking the following checkboxes, it doesn't really test if they are working. We should close the flash.

Reply:
True, I changed it now.

Reply:
I think you forgot to push the changes.

Reply:
Now:laughing:

Reply:
We should be more specific as there might be other buttons with the `data-dismiss` attribute. I suggest:

`find('#flash button[data-dismiss]').click`

Reply:
Hm, if there are multiple ones capybara would fail with some "ambiguous element" error. But I can change it. Might even be that the selector is a bit faster with using the html id

Reply:
Oh right, it would have failed with the `ambiguous element` error.